mason/search/stocks.mas

```diff
@@ -312,21 +312,30 @@ my $breeding_programs_select = simple_selectbox_html(
 var stock_table;
 
 jQuery(document).ready(function () {
+    
+    // I think is unnecessary that jquery function when all fields are reset below
+    // Also returns a jquery error is some cases.
+    // window.addEventListener('load', () => {
+      // // Reset the form parameters
+      // jQuery('#stock_search_form').reset();
```

Comment:
Jquery reset() method is changed now to similar as in the next lines iteration over inputs and selects to set values of them "on load" to empty in advanced search form elements. 
That .reset() returned an error because it was called on a div element with "form" class but not form HTML element.